core/store/models/job_spec_v2.go

```diff
@@ -63,11 +63,11 @@ type (
 		EncryptedOCRKeyBundleID                *Sha256Hash    `json:"keyBundleID" toml:"keyBundleID"                 gorm:"type:bytea"`
 		MonitoringEndpoint                     string         `json:"monitoringEndpoint" toml:"monitoringEndpoint"`
 		TransmitterAddress                     *EIP55Address  `json:"transmitterAddress" toml:"transmitterAddress"`
-		ObservationTimeout                     Interval       `json:"observationTimeout" toml:"observationTimeout" gorm:"type:bigint"`
-		BlockchainTimeout                      Interval       `json:"blockchainTimeout" toml:"blockchainTimeout" gorm:"type:bigint"`
-		ContractConfigTrackerSubscribeInterval Interval       `json:"contractConfigTrackerSubscribeInterval" toml:"contractConfigTrackerSubscribeInterval"`
-		ContractConfigTrackerPollInterval      Interval       `json:"contractConfigTrackerPollInterval" toml:"contractConfigTrackerPollInterval" gorm:"type:bigint"`
-		ContractConfigConfirmations            uint16         `json:"contractConfigConfirmations" toml:"contractConfigConfirmations" default:"3"`
```

Comment:
This `default:"3"` was actually a bug, by _always_ setting it for the config confirmations at the job spec level, you'd always override the global settings, stopping you from actually using the OCR_CONTRACT_CONFIRMATIONS env var.